tests/main/security-device-cgroups-serial-port/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Ensure that the device cgroup works properly for serial-port.
+
+# We don't run the native kernel on these distributions yet so we can't
+# load kernel modules coming from distribution packages yet.
+systems: [-fedora-*, -opensuse-*, -debian-unstable-*]
+
+prepare: |
+    # create serial devices if they don't exist
+    if [ ! -e /dev/ttyS4 ]; then
+        mknod /dev/ttyS4 c 4 68
+        touch /dev/ttyS4.spread
+    fi
+
+restore: |
+    if [ -e /dev/ttyS4.spread ]; then
```

Comment:
The if statement can be dropped, I think.

Reply:
This reasoning is described here: https://github.com/snapcore/snapd/pull/4114#discussion_r148523122